app/views/conditions/dashboard.erb

```diff
@@ -0,0 +1,109 @@
+<h1>conditions dashboard</h1>
+<a href="/conditions" style="color:white;font-size:25px;">See All Conditions</a>
+<div>
+  <table class="table">
+    <p class="station_title"> 
+      <% if @dynamic.nil? %>
+        <%= "create a custom table" %>
+      <% else %>
+        breakdown of <%= @params["attribute"] %> in <%= @params["increment"] %> chunks
+      <% end %>
+    </p> 
+    <div class="query">
+      <form action="/condition-dashboard" method="GET">
+        <span>breakdown by</span> 
+        <select name="attribute">
+          <option value="max_temperature_f">max temp</option>
+          <option value="mean_temperature_f">mean temp</option>
+          <option value="min_temperature_f">min temp</option>
+          <option value="mean_humidity">mean humidity</option>
+          <option value="mean_visibility_miles">mean visibility</option>
+          <option value="max_wind_speed_mph">max wind speed</option>
+        </select> 
+        <span>in</span>
+        <select name="increment">
+          <option value="0.5">.5</option>
+          <option value=1>1</option>
+          <option value=5>5</option>
+          <option value=10>10</option>
+        </select>
+        <span>chunks</span>
+        <input class="custom_button" type="submit" value="fetch">
+      </form>
+    </div>
+        <tr> <th> range </th> <th> average </th> <th> max </th> <th> min </th> </tr>
+        <% unless @dynamic.nil? %>
+          <% @dynamic.each do |k, v| %>
+            <tr>
+              <td> <%= k %> </td> <td> <%= v[0] %> </td> <td> <%= v[1] %> </td> <td> <%= v[2] %> </td> 
+            </tr>
+          <% end %> 
+        <% end %>
+  </table>
+</div>
+<br>
+<div>
+  <table class="table"><p class="station_title"> breakdown by high temperature in 10 degree chunks </p> 
+        <tr> <th> range </th> <th> average </th> <th> max </th> <th> min </th> </tr>
+        <% Condition.ranges_with_trips(:max_temperature_f, 10).each do |k, v| %>
+          <tr>
+            <td> <%= k %> </td> <td> <%= v[0] %> </td> <td> <%= v[1] %> </td> <td> <%= v[2] %> </td> 
+          </tr>
+        <% end %> 
+  </table>
+</div>
+<br>
+<div>
+  <table class="table"><p class="station_title"> breakdown by precipitation in half degree chunks </p> 
+        <tr> <th> range </th> <th> average </th> <th> max </th> <th> min </th> </tr>
+        <% Condition.ranges_with_trips(:precipitation_inches, 0.5).each do |k, v| %>
```

Comment:
same as above. we don't want our views to overstep our controllers. our controller is responsible for talking to our models/database, not our views 👍